models/publisher.js

```diff
@@ -1,41 +1,17 @@
 const mongoose = require('mongoose')
 
 const PublisherSchema = mongoose.Schema({
-    Name :{
-        type:String ,
-        required : true ,
-        trim : true
-    },
-    Email : {
-        type : String,
-        unique : true ,
-        required : true ,
-        trim : true ,
-        lowercase : true
-    },
     mobileNumber :{
         type : String,
         required :true,
-        trim:true,
-        minlength : 10,
-        maxlength : 10
-    },
-    Password :{
-        type : String,
-        required : true ,
-        minlength : 7 ,
-        trim : true 
-    },
-    Gender :{
-        type : String,
-        required:true,
         trim:true
     },
-    Country :{
+    OfficeName :{
```

Comment:
Does it make more sense to make this an array with name `OfficeLocations` and have it contain location & phone number of the publisher offices if they have branches in multiple countries?